cli.js

```diff
@@ -54,6 +54,6 @@ Promise.try(() => {
   }
 
   return ghContrib(cli.input[0], cli.flags)
-}).then(function (response) {
+}).map(function (response) {
```

Comment:
`map` makes no sense, you are not changing anything. You could use `.each` if you want to iterate over the results.